### PR TITLE
feat: add recent argument parameter for transform

### PR DESCRIPTION
**Pull request by Dark0096**
**Background**
- In the case of Counter type metric, converting to 0 when using transformNull can cause an alert. This is because we usually use nonNegativeDerivative to process it. However, if a null value is transformed through the most recent value, the value can be processed without a problem.

`transformNull(seriesList, 'recent')`

1. Origin
[10.0, null, 12.0, null]

2. Transform
[10.0, 10.0, 12.0, 12.0]

**Feature**
- transformNull - support 'recent' argument for Counter type metric

### CODE REVIEW

**Comment by Dark0096** [comment]
For functions with different Graphite specifications, I am worried about where to add the function document.
Do you have a good idea? :(

**Review by jerome89 — COMMENTED** [review]
@Dark0096 
I think it will be fine to put those things in README.md. The default behavior is almost the same as Graphite, so I think there's not many cases to describe the differences between them. We can even create a github page to describe the differences, but seems like it's too much at this point.

**Review by jerome89 — APPROVED** [review]
Thanks for the new feature. It will be useful for us to avoid false alerts.

File: graphene-function/src/main/kotlin/com/graphene/reader/graphite/functions/TransformNullFunction.kt [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  private fun transformNullUsingNumericValue(transform: Double, processedArguments: MutableList<TimeSeries>) {
+    val length = processedArguments[0].values.size
+
+    for (ts in processedArguments) {
+      for (i in 0 until length) {
+        ts.values[i] = if (ts.values[i] != null) ts.values[i] else transform
+      }
+      ts.name = "transformNull(" + ts.name + "," + (if (DoubleMath.isMathematicalInteger(transform)) Integer.toString(transform.toInt()) else transform) + ")"
```
[the remark this answers is not in the document]
Comment (Dark0096) [reply graphene-function/src/main/kotlin/com/graphene/reader/graphite/functions/TransformNullFunction.kt]: Thanks for your advice. Fix it!

**Comment by Dark0096** [comment]
:)

**Pull request merged**